schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/ConfigEntry.java

```diff
@@ -128,6 +137,44 @@ public String toString() {
                 '}';
     }
 
+    /**
+     * Parses a String to a typed value
+     *
+     * @param <T> type of the resulting value.
+     */
+    public interface Converter<T> {
+        T convert(String str);
+    }
+
+    public static class IntegerConverter implements Converter<Integer> {
+
+        private static Converter<Integer> instance = new IntegerConverter();
```

Comment:
Can we make this **private static final**  ?

Reply:
Good point, I'm going to change it.